web/controllers/rules/pr/merge_conflict.ex

```diff
@@ -7,7 +7,8 @@ defmodule Dwylbot.Rules.PR.MergeConflict do
   @rule_name "pr_merge_conflicts"
 
   def apply?(payload) do
-    payload["action"] == "closed" && payload["pull_request"]["merged"]
+    (payload["action"] == "closed" && payload["pull_request"]["merged"])
+    || (payload["action"] == "labeled" && payload["label"]["name"] == "awaiting-review")
```

Comment:
It might be good to add a test for this function. I think a `doctest` might be good here.

I'm not sure if using multiple `||` and `&&`s are all that readable. Perhaps something like this would be better:

```elixir
def apply?(%{"action" => "closed", "pull_request" => %{"merged" => merged}}), do: merged
def apply?(%{"action" => "labeled", "label" => %{"name" => "awaiting-review"}}), do: true
```

Reply:
Thanks @shouston3. Since adding your suggestions, I've been getting the following error:
<img width="757" alt="screen shot 2017-09-01 at 11 30 53" src="https://user-images.githubusercontent.com/11833296/29966266-0d713d2c-8f09-11e7-87fd-55137d7adfd0.png"> 😕